kratos/geometries/geometry.h

```diff
@@ -733,6 +734,54 @@ class Geometry
     ///@name Inquiry
     ///@{
 
+    /**
+     * @brief Checks if two GeometryType have the same type
+     * @return True if the objects are the same type, false otherwise
+     */
+    inline static bool HasSameType(const GeometryType& rLHS, const GeometryType& rRHS) {
```

Comment:
```suggestion
    inline static bool HasSameType(
        const GeometryType& rLHS,
        const GeometryType& rRHS)
    {
```
(minor style guide comment)